programs/client/clickhouse-client.xml

```diff
@@ -1,5 +1,6 @@
 <!-- Config set into /etc/clickhouse-client/. It's used if no other configs are found. -->
 <config>
+    <!-- Shorthand for self-signed combination in openSSL section below: <accept-invalid-certificate>1</accept-invalid-certificate> -->
```

Comment:
Should we add `<accept-invalid-certificate>` as a commented tag to below code (similar to l. 12). Most users don't read the source code and will otherwise have no idea that `<accept-invalid-certificate>` exists.

Reply:
what do you mean? it is a commented tag exactly as at lines 10 and 12